src/main/java/seedu/address/model/ingredient/IngredientQuantity.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.ingredient;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * A class to represent the ingredientQuantity in an ingredient
+ */
+
+public class IngredientQuantity {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Ingredient's quantity should be non-zero unsigned integer.";
+
+    // Identity fields
+    private int ingredientQuantity;
+    /**
+     * Constructs a {@code IngredientQuantity}.
+     *
+     * @param quantity A valid quantity, an integer that is non-negative.
+     */
+    public IngredientQuantity(int quantity) {
+        this.ingredientQuantity = quantity;
+    }
+
+    public int getQuantity() {
+        return ingredientQuantity;
+    }
+
+    /**
+     * Returns true if ingredientQuantity is valid.
+     * @param test
+     * @return
```

Comment:
javadoc: don't have a blank '@return'. if you dont want to put anything, leave it blank